### PR TITLE
[Resolves #7619] Align geostory and fix issue with the thesauri

### LINKED ISSUE #7619: GeoStory: error in metadata wizard
## Expected Behavior
Once the thesauri are configured in geonode, for the GeoApps, once i save the metadata, the save should go smoothly

## Actual Behavior
If i save the GeoApps, it throws an error regarding the missing thesauro
immagine

## Steps to Reproduce the Problem

  1. load a thesauro
  2. create a geostory app
  3. try to save

## Specifications

  - GeoNode version: master, 3.2.x
  - Installation method (manual, GeoNode Docker, SPCGeoNode Docker):
  - Platform:
  - Additional details:

### PR DESCRIPTION
**Pull request by mattiagiupponi**
References #7619

- fix thesauri in geostory
- align geostory with the other templates format

## Checklist

> Reviewing is a process done by project maintainers, mostly on a volunteer basis. We try to keep the overhead as small as possible and appreciate if you help us to do so by completing the following items. Feel free to ask in a comment if you have troubles with any of them.

For all pull requests:

- [x] Confirm you have read the [contribution guidelines](https://github.com/GeoNode/geonode/blob/master/CONTRIBUTING.md) 
- [x] You have sent a Contribution Licence Agreement (CLA) as necessary (not required for small changes, e.g., fixing typos in the documentation)
- [x] Make sure the first PR targets the master branch, eventual backports will be managed later. This can be ignored if the PR is fixing an issue that only happens in a specific branch, but not in newer ones.

The following are required only for core and extension modules (they are welcomed, but not required, for contrib modules):
- [x] There is a ticket in https://github.com/GeoNode/geonode/issues describing the issue/improvement/feature (a notable exemption is, changes not visible to end-users)
- [x] The issue connected to the PR must have Labels and Milestone assigned
- [x] PR for bug fixes and small new features are presented as a single commit
- [x] Commit message must be in the form "[Fixes #<issue_number>] Title of the Issue"
- [ ] New unit tests have been added covering the changes, unless there is an explanation on why the tests are not necessary/implemented
- [ ] This PR passes all existing unit tests (test results will be reported by travis-ci after opening this PR)
- [x] This PR passes the QA checks: flake8 geonode
- [ ] Commits changing the **settings**, **UI**, **existing user workflows**, or adding **new functionality**, need to include documentation updates
- [ ] Commits adding **new texts** do use gettext and have updated .po / .mo files (without location infos)

**Submitting the PR does not require you to check all items, but by the time it gets merged, they should be either satisfied or inapplicable.**

### CODE REVIEW

**Comment by afabiani** [comment]
@mattiagiupponi please fix pep8 issues

```
^@^@geonode/geoapps/forms.py:35:1: W293 blank line contains whitespace
geonode/geoapps/views.py:389:35: E272 multiple spaces before keyword
```

File: geonode/geoapps/forms.py [hunk 1]
```diff
@@ -32,3 +32,20 @@ class Meta(ResourceBaseForm.Meta):
             'center_y',
             'data'
         )
+    
+    def __init__(self, *args, **kwargs):
```
Comment (afabiani) [inline geonode/geoapps/forms.py:36]: Is this necessary?

Comment (mattiagiupponi) [reply geonode/geoapps/forms.py:36]: It's required to correctly show the tooltips in metadata editing like:

image

**Comment by mattiagiupponi** [comment]
> @mattiagiupponi please fix pep8 issues
> 
> ```
> ^@^@geonode/geoapps/forms.py:35:1: W293 blank line contains whitespace
> geonode/geoapps/views.py:389:35: E272 multiple spaces before keyword
> ```

Fixed

**Review by afabiani — APPROVED** [review]

**Pull request merged**